src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -237,6 +237,12 @@ and whether it is an `isPrimary` one. Note that the built-in screen
 The `currentScreen` field is a live object corresponding to the current `window.screen`. The object
 is updated on cross-screen window placements or device changes.
 
+{% Aside %}
+Earlier versions of the API would not return detailed screen labels like `"Built-in Retina
```

Comment:
Consider rephrasing to something like: "Chrome's latest API implementation returns screen labels provided by the underlying platform, like `"Built-in Retina Display"`, instead of less descriptive placeholders, like `"Internal Display 1"`.